recipes/vscode-jupyter/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs 12
+    - code-server >=3.9
```

Comment:
They shouldn't be in `build`
```suggestion
```

Reply:
When removing those two build requirements, [the pipelines fails](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=285286&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823)

```sh
Error: Cannot find module '/home/conda/staged-recipes/build_artifacts/vscode-jupyter_1614789822797/_build_env/share/code-server/out/node/entry.js'
```

I'm a pretty sure that `code-server` is required in the build environment (why I don't know). I'm testing this locally now.

Reply:
https://github.com/conda-forge/staged-recipes/pull/14117/commits/200a1f81ca9e4b1457c611a18263e6fcf78a602f solved this issue ([the pipeline still fails](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=285596&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823) but only because of a failed test).

Reply:
Needs https://github.com/conda-forge/code-server-feedstock/pull/42

Reply:
Thanks!

[The Linux build still fails because of some compiler issues that I don't really understand.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=285808&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1394)

Reply:
I think we should package tools like `gulp` or `vsce` in conda-forge as they are otherwise included in this package. The Linux build error though is about pre-built binaries that come with the `zeromq.js` package. We should built that instead with the conda-forge provided `libzmq`.